app/javascript/flavours/glitch/actions/interactions.js

```diff
@@ -392,3 +402,78 @@ export function unpinFail(status, error) {
     error,
   };
 };
+
+export const addReaction = (statusId, name) => (dispatch, getState) => {
+  const status = getState().get('statuses').get(statusId);
+  let alreadyAdded = false;
+  if (status) {
+    const reaction = status.get('reactions').find(x => x.get('name') === name);
+    if (reaction && reaction.get('me')) {
+      alreadyAdded = true;
+    }
+  }
+  if (!alreadyAdded) {
```

Comment:
What does alreadyAdded actually do?
It's passed to addReactionRequest and addReactionSuccess, but isn't used.
Am I missing something?

Reply:
This must be a leftover from `announcements.js` in the same directory because that's what where I took the code from originally.  It should be safe to delete.